app/Http/Controllers/WebhookController.php

```diff
@@ -83,9 +89,26 @@ public function edit(Webhook $webhook)
      * @param  \App\Models\Webhook  $webhook
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, Webhook $webhook)
+    public function update(WebhookUpdateRequest $request)
     {
-        //
+        $data = $request->except('_token');
+        $response = $this->webhookRepository->getWebhookByName($data['name']);
+        if(isset($response)) {
+            if($response->id != $data['id']){
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 97</br>Blank line found at start of control structure